graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/MessagesAdapterES7.java

```diff
@@ -251,7 +252,7 @@ private IndexingError.Type errorTypeFromResponse(BulkItemResponse item) {
     private IndexRequest indexRequestFrom(IndexingRequest request) {
         final byte[] body;
         try {
-            body = this.objectMapper.writeValueAsBytes(request.message().toElasticSearchObject(objectMapper, this.invalidTimestampMeter));
+            body = request.message().serialize(new DefaultSerializationContext(objectMapper, this.invalidTimestampMeter));
```

Comment:
How about adding a static `SerializationContext.of()` method to the interface to avoid using `DefaultSerializationContext`?